src/test/java/com/conveyal/gtfs/GTFSTest.java

```diff
@@ -627,8 +809,15 @@ private void assertThatImportedGtfsMeetsExpectations(
                     LOG.error("Persistence mismatch on record {}", numRecordsSearched);
                 }
             }
-            assertThatPersistenceExpectationRecordWasFound(numRecordsSearched, foundRecord, fieldsWithMismatches);
+            assertThatDatabasePersistenceExpectationRecordWasFound(
+                persistenceExpectation,
+                numRecordsSearched,
+                foundRecord,
+                fieldsWithMismatches
+            );
         }
+        // Skip error expectation analysis on editor database
+        if (editorDatabase) return;
```

Comment:
This seems risky. What if something were added to the end of this function that pertained to both editor and non-editor DBs?